kubeshop/botkube
# Add documentationURL to plugins

## Description

Changes proposed in this pull request:

- Add Documentation URL to plugins

## Testing


- rebuild your local plugins  `make build-plugins-single`
- run your local plugin index server `go run test/helpers/plugin_server.go`
- open browser on (in my case it was http://localhost:3010/botkube.yaml)
- check if you can see `documentationUrl` fill in with the link to our docs for each plugin

## Related issue(s)


https://github.com/kubeshop/botkube-cloud/issues/649
